packit_service/worker/github_handlers.py

```diff
@@ -294,6 +296,19 @@ def handle_pull_request(self):
         return handler_results
 
     def run(self) -> HandlerResults:
+        is_copr_build: Callable[
+            [JobConfig], bool
+        ] = lambda job: job.type == JobType.copr_build
+
+        if self.job.job == JobType.tests and any(
+            filter(is_copr_build, self.package_config.jobs)
+        ):
```

Comment:
I'd write this with a for loop, or at least pass the lambda directly to the filter call

but if you and Jirka think this is fine, let's roll

Reply:
It's (currently) impossible to type lambda directly... I would be glad to see a better way to satisfy `mypy`...